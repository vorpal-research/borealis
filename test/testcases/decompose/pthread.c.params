---config:test/testcases/decompose/wrapper.decompose.conf
