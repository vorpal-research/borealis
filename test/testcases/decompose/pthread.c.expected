[
	{
		"location" : 
		{
			"filename" : "test/testcases/decompose/pthread.c",
			"loc" : 
			{
				"col" : 1,
				"line" : 51
			}
		},
		"type" : "ASR-01"
	}
]
